nbs/examples/ElectricityPeakForecasting.ipynb

```diff
@@ -0,0 +1,14449 @@
+{
```

Comment:
<p>Here it might be useful to point out that StatsForecas has a wide variety of models that can be found in the documentation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I added it in the cell below.</p>